day1OfJS/variable.js

```diff
@@ -0,0 +1,65 @@
+/*
+Create a variable.js file and declare variables and assign string, boolean, undefined and null data types
+
+let vr = "javascript";
```

Comment:
As a rule of thumb: if you are declaring _variables_, start by using `const` instead of `let` or `var`, and if you need to reassign values later, then that's when you know you need to use `let`. So, let's update all the variables declared in this exercise to use `const` for those that apply.

Reply:
Changed variables - 4f7f6f6